OpenRA.Game/Map/Map.cs

```diff
@@ -483,10 +483,11 @@ public void Save(string toPath)
 				}
 			}
 
-			// Saving the map to a new location
-			if (toPath != Path)
+			if (toPath != Path || Container == null)
 			{
-				Path = toPath;
+				// Saving the map to a new location
+				if (Path != toPath)
```

Comment:
This check is unnecessary.  If `Path == toPath` then reassigning the variable doesn't change anything.